functions/src/index.ts

```diff
@@ -2,32 +2,59 @@ import * as functions from "firebase-functions";
 import { store } from "./database";
 
 export const save = functions.https.onRequest((request, response) => {
-  const event = {
-    // Client-provided values
-    uid: request.body.uid,
-    client: request.body.client,
-    key: request.body.key,
-    value: request.body.value,
+  const body = request.body;
+  console.log(`Body = ${JSON.stringify(body)}`);
```

Comment:
This is useful for debugging, but I'd remove this for production, since it will result in a lot of logs with unclear retention policies.